app/src/main/res/values-qq/strings.xml

```diff
@@ -1087,4 +1087,5 @@
   <string name="watchlist_page_moved">Label text to indicate that a page has been moved.</string>
   <string name="watchlist_page_protected">Label text to indicate that a page has been protected.</string>
   <string name="watchlist_page_deleted">Label text to indicate that a page has been deleted.</string>
+  <string name="add_app_language_text">Label text to add or edit apps langugaes list.</string>
```

Comment:
Would it be possible to have the string name has a `notification_filters` prefix in it?

Reply:
I added it to the `Multilingual` section so it could be used other places too, so no prefix would keep it generic... what do you think?

Reply:
I recall there was a ticket about *not* recommending using the same strings because of countries/geolocations.